Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM metabrainz/consul-template-base:v0.18.5-1
+
+ARG BUILD_DEPS=" \
```

Comment:
The reason I originally separated these from `RUN_DEPS` was to uninstall them from the image after they weren't needed anymore, e.g.: https://github.com/metabrainz/CAA-indexer/blob/a8a27df1b1e2f8961b2e0cdae63231b552dd22e2/Dockerfile#L44-L44 though it's not really necessary, just reduces the image size a bit.

Reply:
Ahh, yep. Meant to remove this; I'll address this.